examples/rust/components/blobby/src/wrapper.rs

```diff
@@ -0,0 +1,112 @@
+// This is adapted from `wasmcloud-actor` but instead is impl'd directly on the type. This is
```

Comment:
Why not use `wasmcloud_actor` adapters directly?

That's the only necessary change to implementation to make it work: https://github.com/wasmCloud/wasmCloud/blob/ffad11e8caf1fe39aead4ed9843bfd21b1c248ba/tests/actors/rust/builtins-component-reactor/src/lib.rs#L6-L8

Reply:
Wasmcloud actor isn't published and I expect people to work from this as a starting point from their own applications